tests/cross/src/test/scala/tests/pc/CompletionSuite.scala

```diff
@@ -1015,8 +1005,8 @@ class CompletionSuite extends BaseCompletionSuite {
     topLines = Some(2),
     compat = Map(
       "3" ->
-        """|NoManifest scala.reflect
-           |NoClassDefFoundError java.lang
+        """|None scala
+           |Some(value) scala
```

Comment:
```suggestion
```
Some shouldn't be suggested since we already wrote `No`

Reply:
Right, it's because we don't filter completions by prefix while creating them and leave it to the editor. I have an idea how to fix it